nav2_bringup/bringup/launch/tb3_simulation_launch.py

```diff
@@ -130,8 +130,8 @@ def generate_launch_description():
         # TODO(orduno) Switch back once ROS argument passing has been fixed upstream
         #              https://github.com/ROBOTIS-GIT/turtlebot3_simulations/issues/91
         # default_value=os.path.join(get_package_share_directory('turtlebot3_gazebo'),
-        #                            'worlds/turtlebot3_worlds/waffle.model'),
-        default_value=os.path.join(bringup_dir, 'worlds', 'waffle.model'),
+        #                            'worlds/turtlebot3_worlds/house.world'),
+        default_value=os.path.join(bringup_dir, 'worlds', 'house.world'),
```

Comment:
We don't have a house.world file in our package for that to be a valid default. If you'd like to propose adding one, we can talk about that for sure, but in another ticket / PR

Reply:
Looks like my IDE did some automatic refactor with cached information about a house world I worked on for a bit. My bad for not checking.